### PR TITLE
Papercut: Introduce static final field NEW_LINE and use it instead of getLineSeperator()

### PR DESCRIPTION
**Pull request by jerrinot**
The field is slightly less awkward to use, it's a standard practice in other projects
and it could have performance benefits as well.

### CODE REVIEW

File: hazelcast/src/main/java/com/hazelcast/internal/monitors/HealthMonitor.java [hunk 1]
```diff
@@ -154,7 +154,7 @@ private void logPerformanceMonitorHint() {
 
             logger.info(String.format("The HealthMonitor has detected a high load on the system. For more detailed information,%s"
                             + "enable the PerformanceMonitor by adding the property -D%s=true",
-                    getLineSeperator(), GroupProperty.PERFORMANCE_MONITOR_ENABLED));
+                    NEW_LINE, GroupProperty.PERFORMANCE_MONITOR_ENABLED));
```
Comment (Donnerbart) [inline hazelcast/src/main/java/com/hazelcast/internal/monitors/HealthMonitor.java]: In this case you can just use `%n` in the text, if I'm not wrong.

Comment (jerrinot) [inline hazelcast/src/main/java/com/hazelcast/internal/monitors/HealthMonitor.java]: correct

**Comment by Donnerbart** [comment]
You could use the constant in the deprecated method itself as well to save a bit of performance if it's still used somewhere. But that's just nitpicking :)

:+1: (just minor comments)

**Comment by jerrinot** [comment]
@Donnerbart: thanks, the PR is updated.

**Comment by jerrinot** [comment]
run-lab-run

**Comment by jerrinot** [comment]
run-lab-run

**Comment by pveentjer** [comment]
Please check StringUtils. LINE_SEPARATOR

**Comment by jerrinot** [comment]
I'm closing this one as @pveentjer introduced the `LINE_SEPARATOR` in a different PR.

**Pull request closed without merging**